flexibleengine/resource_flexibleengine_cce_cluster_v3.go

```diff
@@ -289,13 +289,17 @@ func resourceCCEClusterV3Read(d *schema.ResourceData, meta interface{}) error {
 	d.SetId(n.Metadata.Id)
 	d.Set("name", n.Metadata.Name)
 	d.Set("status", n.Status.Phase)
+	d.Set("labels", n.Metadata.Labels)
+	d.Set("annotations", n.Metadata.Annotations)
 	d.Set("flavor_id", n.Spec.Flavor)
 	d.Set("cluster_type", n.Spec.Type)
+	d.Set("cluster_version", n.Spec.Version)
 	d.Set("description", n.Spec.Description)
 	d.Set("billing_mode", n.Spec.BillingMode)
 	d.Set("vpc_id", n.Spec.HostNetwork.VpcId)
 	d.Set("subnet_id", n.Spec.HostNetwork.SubnetId)
 	d.Set("highway_subnet_id", n.Spec.HostNetwork.HighwaySubnet)
+	d.Set("extend_param", n.Spec.ExtendParam)
```

Comment:
Seems `extend_param` responed with some extra informations, so this will make the next `plan` not empty.

Reply:
i removed it. But FE offers the possibility to build a multi AZ kubernetes API via extend_param, so we will need a way discover this attribute to import an unmanaged cluster

https://docs.prod-cloud-ocb.orange-business.com/api2/cce/cce_02_0236.html#cce_02_0236__table17575013586